### ISSUE TITLE
Battery Optimizations

### ISSUE THREAD

**Issue by machiav3lli** [fennifith/Alarmio#37]
Hello again, just wanted to point out, that with some more conservative cpu profiles, the app does miss some of the alarms which results,-you can guess- "missing the lecture" issue, I've tested it with some other clock-apps and noted that it didn't happen, secondly I disabled the battery optimizations of the app, hoping to avoid this issue, it worked so far, will report further. Maybe this should be implemented(asking to disable the battery optimizations at start).
Thanks for your work.

**Comment by machiav3lli** [comment]
@fennifith yeah, it's totaly something else and am on AOSP based CustomRom, so it's not RAM management problem, I even live the app in the recent apps, and take a 30 minutes nap(with wifi-off), it misses some of the alarms

**Comment by fennifith** [comment]
There is now an option for ignoring the battery optimizations at, um, the bottom of the settings menu. No UI stating why this option is there or anything. I guess I'll implement that later.
